storage_test.go

```diff
@@ -0,0 +1,11 @@
+package tprstorage
+
+import (
+	"testing"
+
+	"github.com/giantswarm/microstorage"
+)
+
+func TestInterface(t *testing.T) {
```

Comment:
Yeah we could test slashes, but why to do this sort of exercise when the real test isn't in place. Things like that should be covered in integration tests anyway....